JS-FromFundamentalsToFunctionalJsV2/7.callbacks/_reduce.js

```diff
@@ -0,0 +1,9 @@
+// import lodash
+const _ = require('../shared/lodash');
+const _lib = require('lodash');
```

Comment:
I think it will be better if you rename _ with lodash to avoid confusing with underscore lib